linux/Dockerfile

```diff
@@ -19,6 +21,38 @@ RUN apt-get update && \
 	curl \
 	jq
 
+# --- kcov --- #
+# Install dependencies
+RUN apt-get install -y \
+	cmake \
+	libcurl4-openssl-dev \
+	libelf-dev \
+	libdw-dev \
+	gcc \
+	binutils-dev \
+	zlib1g-dev \
+	libiberty-dev \
+	pkg-config \
+	python2.7 \
+	wget \
+	zip
+
+# For some reason `python` isn't the default executable file name.
+# Needed for kcov
+RUN ln -s /usr/bin/python2.7 /usr/bin/python
+
+# Download kcov
+# -nc is okay since we are downloading a tagged version
+RUN wget -nc https://github.com/SimonKagstrom/kcov/archive/v$KCOV_VERSION.zip
+RUN unzip -uoq v$KCOV_VERSION.zip
```

Comment:
To avoid having zip in the image, combine above layers and delete the file in one go
```dockerfile
 RUN wget -nc https://github.com/SimonKagstrom/kcov/archive/v$KCOV_VERSION.zip -O kcov.zip && \
     unzip -uoq kcov.zip && \
     rm kcov.zip
```

Reply:
ah true -- didn't realize having a pristine image was important 👌